src/components/navBar/navBar.css

```diff
@@ -0,0 +1,96 @@
+.navBar-container {
+    display: flex;
+    flex-direction: row;
+    width: 1440px;
```

Comment:
In components like a Navbar, you need to span the entire width of the viewport, using fixed width will be detrimental when styling responsive components. If viewport will be wider than 1440px your Navbar will not span the entire screen.
Use **width: 100%** instead